.github/workflows/downloadable_artifact.yml

```
@@ -0,0 +1,17 @@
+name: Generate Word docs
+on: push
+
+jobs:
+  convert_via_pandoc:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: convert md to docx
+        uses: docker://pandoc/latex:2.9
+        with:
+          entrypoint: /bin/sh
+          args: -c "mkdir output;for d in */; do mkdir output/$d; done;find ./ -iname '*.md' -type f -exec sh -c 'pandoc ${0} -o output/${0%.md}.docx' {} \;"
```

There is another way to do this which might be a bit more understandable:

```suggestion
    container:
      image: docker://pandoc/latex:2.9
      options: --entrypoint=sh
    steps:
      - uses: actions/checkout@v2
      - name: prepare output directories
        run: |
          for d in */; do
            mkdir -p output/$d
          done
      - name: convert md to docx
        run: |
          find ./ -iname '*.md' -type f -exec sh -c 'pandoc ${0} -o output/${0%.md}.docx' {} \;
```

Same overall effect, but being able specify a bunch of steps to run inside the container and hence break up the scripting into logical blocks just makes for a nicer UX in my opinion. It might not be a big deal at the level of complexity of this shell command string, but as things grow into more complicated scenarios where maybe you wanted to run a bunch of Pandoc commands or add filters etc. this way will be more manageable.

---

Much better!

---

🏊🏻